.travis.yml

```diff
@@ -15,8 +15,8 @@ env:
   - JOB=test
 
 script:
-  - 'if [ "$JOB" = "test" ]; then make test-ci; fi'
-  - 'if [ "$JOB" = "test-coverage" ]; then make test-ci-coverage; fi'
+  - 'if [ "$JOB" = "test" ]; then CI=true make test-ci; fi'
```

Comment:
Oh! Thanks, they have to run sequentially because Travis machine resources are limited, by default Jest spawn child processes that consume more resources